app/create_file.py

```diff
@@ -1 +1,54 @@
-# write your code here
+import sys
+import os
+from datetime import datetime
+
+
+def create_file() -> None:
+    arguments = sys.argv[1:]
+    file_name = "file.txt"
+    directories = []
+    if "-f" in arguments:
+        if "-d" in arguments and "-f" in arguments:
```

Comment:
you already checked it 2 lines above
```suggestion
        if "-d" in arguments:
```